app/src/androidTest/java/com/github/brugapp/brug/ChatMenuActivityTest.kt

```diff
@@ -0,0 +1,186 @@
+package com.github.brugapp.brug
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.GeneralLocation
+import androidx.test.espresso.action.GeneralSwipeAction
+import androidx.test.espresso.action.Press
+import androidx.test.espresso.action.Swipe
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.contrib.RecyclerViewActions
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.Intents.intended
+import androidx.test.espresso.intent.matcher.IntentMatchers.hasComponent
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.UiDevice
+import androidx.test.uiautomator.UiScrollable
+import androidx.test.uiautomator.UiSelector
+import org.hamcrest.core.IsEqual
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.io.IOException
+
+private const val DUMMY_TEXT: String = "Actual behavior coming soon…"
+private const val DELETE_CHAT_TEXT = "Chat feed has been deleted."
+private const val APP_PACKAGE_NAME: String = "com.github.brugapp.brug"
+
+private const val LIST_VIEW_ID: String = "$APP_PACKAGE_NAME:id/chat_listview"
+private const val LIST_ENTRY_ID: String = "$APP_PACKAGE_NAME:id/chat_entry_title"
+private const val SNACKBAR_ID: String = "$APP_PACKAGE_NAME:id/snackbar_text"
+
+@RunWith(AndroidJUnit4::class)
+class ChatMenuActivityTest {
+
+    @get:Rule
+    val testRule = ActivityScenarioRule(ChatMenuActivity::class.java)
+
```

Comment:
Here I would suggest putting Intents.init() and Intents.release() in @Before and @After  annotated functions. Since you have multiple tests that use intents, it avoids conflicts with init() and release() between tests and code duplication.
```suggestion
@Before
fun setUp() {
    Intents.init()
}

@After
fun finishUp() {
    Intents.release()
}
```

Reply:
Thank you for your suggestion, indeed it seems to be a better way to manage the Intents in my tests, and it avoids unnecessary code duplication. This will be fixed in the next commit.